primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Dialogue: 0,0:11:40.95,0:11:46.48,Default,,0,0,0,,As it develops, its intelligence \Nwill surpass that of a human.
+Dialogue: 0,0:11:46.48,0:11:48.38,Default,,0,0,0,,Its physical prowess will follow suit.
 Dialogue: 0,0:11:50.49,0:11:54.62,Default,,0,0,0,,However, its true latent capabilities are unknown.
 Dialogue: 0,0:11:54.62,0:11:57.50,Default,,0,0,0,,Strict supervision will be required.
 Dialogue: 0,0:11:58.50,0:12:00.44,Default,,0,0,0,,It is currently docile.
-Dialogue: 0,0:12:00.44,0:12:06.04,Default,,0,0,0,,But if one day they were to turn against us...{lit bare fangs, which is of course an idiom in English, but I dunno if it sounds right for this character}
-Dialogue: 0,0:12:12.83,0:12:17.05,Default,,0,0,0,,Captain, how long will I be able to live?
+Dialogue: 0,0:12:00.44,0:12:06.04,Default,,0,0,0,,But if one day it were to turn against us...{lit bare fangs, which is of course an idiom in English, but I dunno if it sounds right for this character}
+Dialogue: 0,0:12:12.83,0:12:17.05,Default,,0,0,0,,Captain, how long do I have to live?
 Dialogue: 0,0:12:17.56,0:12:19.88,Default,,0,0,0,,Are you afraid of dying?
 Dialogue: 0,0:12:22.17,0:12:23.38,Default,,0,0,0,,Not at all.
-Dialogue: 0,0:12:23.38,0:12:27.00,Default,,0,0,0,,It's just... I'm an artificial being.
-Dialogue: 0,0:12:27.00,0:12:29.23,Default,,0,0,0,,My lifespan is different to yours.
+Dialogue: 0,0:12:23.38,0:12:27.00,Default,,0,0,0,,It's just... I'm a homunculus.
+Dialogue: 0,0:12:27.00,0:12:29.23,Default,,0,0,0,,My lifespan is different than yours.
 Dialogue: 0,0:12:29.23,0:12:30.36,Default,,0,0,0,,So I...
 Dialogue: 0,0:12:31.00,0:12:34.26,Default,,0,0,0,,I want to understand my own life.
-Dialogue: 0,0:12:35.24,0:12:39.01,Default,,0,0,0,,The organisation is responsible for all of your wellbeing.
-Dialogue: 0,0:12:39.01,0:12:40.49,Default,,0,0,0,,There's nothing to be worried about.
-Dialogue: 0,0:12:41.77,0:12:45.46,Default,,0,0,0,,However, we won't know your lifespan without more research.
-Dialogue: 0,0:12:46.75,0:12:50.50,Default,,0,0,0,,As the first generation,\Nyou are the research subjects.
+Dialogue: 0,0:12:35.24,0:12:39.01,Default,,0,0,0,,The organization is responsible \Nfor all of your well-beings.
+Dialogue: 0,0:12:39.01,0:12:40.49,Default,,0,0,0,,Don't stress about it.
+Dialogue: 0,0:12:41.77,0:12:45.46,Default,,0,0,0,,However, we won't know your lifespans \Nwithout more research.
+Dialogue: 0,0:12:46.75,0:12:50.50,Default,,0,0,0,,As the first generation of homunculi, \Nyou girls are the prototypes.
 Dialogue: 0,0:12:51.52,0:12:53.13,Default,,0,0,0,,That's enough pointless chitchat.
 Dialogue: 0,0:12:53.13,0:12:55.49,Default,,0,0,0,,You just need to follow orders.
 Dialogue: 0,0:12:55.49,0:12:56.79,Default,,0,0,0,,All right?
 Dialogue: 0,0:12:56.79,0:12:57.98,Default,,0,0,0,,Understood.
 Dialogue: 0,0:12:59.78,0:13:00.99,Default,,0,0,0,,That's good.
 Dialogue: 0,0:13:00.99,0:13:03.35,Default,,0,0,0,,Your orders are absolute.
```

Comment:
"Absolute" seems a bit stiff, but she's being stiff, too. I'd go with "final" if we had to change it out, but I think we're okay.